editoast/src/generated_data/mod.rs

```diff
@@ -164,18 +166,19 @@ pub mod tests {
     use rstest::rstest;
     use std::ops::DerefMut;
 
-    use crate::fixtures::tests::db_pool;
     use crate::generated_data::clear_all;
     use crate::generated_data::refresh_all;
     use crate::generated_data::update_all;
     use crate::modelsv2::fixtures::create_empty_infra;
     use editoast_models::DbConnectionPoolV2;
 
     #[rstest] // Slow test
+    /// For more details, see `DbConnectionPoolV2::get`, Deadlocks section.
```

Comment:
```suggestion
    // PostgreSQL deadlock can happen in this test, see section `Deadlock` of [DbConnectionPoolV2::get] for more information
```

`///` is for documentation, not comments

When referencing an object, better use `[ns::obj]` to allow the CI check the link isn't broken.

Reply:
Done.